### PR TITLE
Removing zero-division warnings

### PR DESCRIPTION
**Pull request by alliesaizan**
This pull request addresses one of the issues mentioned in #852. There also appear to be minor edits related to black formatting (changing single quotation marks to double quotation marks and changing spaces to tabs for multi-line commands).

### CODE REVIEW

File: examples/plot_new_metrics.py [hunk 1]
```diff
@@ -231,14 +230,20 @@ def occupation_transform(occ_string):
 # class. Let us construct an instance of this class, and then look at
 # its capabilities:
 
-fbeta_06 = functools.partial(skm.fbeta_score, beta=0.6)
+fbeta_06 = functools.partial(skm.fbeta_score, beta=0.6, zero_division=1)
```
Comment (romanlutz) [inline examples/plot_new_metrics.py:233]: This is the only "real" change other than the style fixes, right?

**Review by romanlutz — APPROVED** [review]

File: examples/plot_new_metrics.py (hunk 1 above)
Comment (romanlutz) [reply examples/plot_new_metrics.py:233]: Confirmed offline that this is true. Thanks @alliesaizan!

Comment (riedgar-ms) [reply examples/plot_new_metrics.py:233]: I'm happy with this. Ideally all the formatting changes would be a separate PR...

**Review by riedgar-ms — APPROVED** [review]

**Comment by riedgar-ms** [comment]
The builds are failing due to `flake8` :
/examples/plot_new_metrics.py:166:69: E231 missing whitespace after ','

**Pull request merged**